.github/workflows/run_tests.yml

```diff
@@ -65,7 +67,7 @@ jobs:
 
       - name: Run tests
         run: |
-          pytest
+          pytest -v
```

Comment:
Should we keep this setting for now until we find that the test output is getting too long?

Reply:
yes I thiink so